Curs09/C/02-max_struct.c

```diff
@@ -0,0 +1,79 @@
+﻿/* Programarea Calculatoarelor CA, 2022-2023
+
+	Crearea unei structuri cu 2 campuri (nume si nota) si 
+		afisarea numelui cu cea mai mare nota. 
+
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define NAMES_COUNT 5
+#define MAX_NAME_LENGTH 8
+
+struct Elev
+{
+	char Nume[MAX_NAME_LENGTH];
+	float Nota;
+};
+
+struct Elev* getRandomStudents(const char MulteNume[][MAX_NAME_LENGTH])
```

Comment:
```suggestion
struct Elev* getRandomStudents(const char MulteNume[][MAX_NAME_LENGTH])
```

change MulteNume to something more appropriate. (e.g: students_names).